examples/AMY_MIDI_Synth/AMY_MIDI_Synth.ino

```diff
@@ -50,9 +44,6 @@ void setup() {
   amy_config.has_reverb = 1;
```

Comment:
You got rid of these `has_*` flags too, right?  I guess you didn't test this bit on Arduino?